webui/src/views/ImportView.vue

```diff
@@ -0,0 +1,104 @@
+<script setup>
+import { ref, computed, onMounted } from 'vue'
+import DescriptionPanel from '../components/DescriptionPanel.vue'
+import UploadOptions from '../components/UploadOptions.vue'
+import MetadataPanel from '../components/MetadataPanel.vue'
+import RuleMatchesTable from '../components/RuleMatchesTable.vue'
+import FunctionCapabilities from '../components/FunctionCapabilities.vue'
+import ProcessCapabilities from '../components/ProcessCapabilities.vue'
+import SettingsPanel from '../components/SettingsPanel.vue'
+import Toast from 'primevue/toast'
+
+import demoRdocStatic from '../../../tests/data/rd/al-khaser_x64.exe_.json'
+import demoRdocDynamic from '../../../tests/data/rd/0000a65749f5902c4d82ffa701198038f0b4870b00a27cfca109f8f933476d82.json'
```

Comment:
let's update to a v7 document so this actually works

Reply:
I should also add a path resolving shortcut instead of using `../../`